src/main/java/redis/clients/jedis/exceptions/JedisAuthenticationException.java

```diff
@@ -0,0 +1,16 @@
+package redis.clients.jedis.exceptions;
+
+public class JedisAuthenticationException extends JedisException {
```

Comment:
There should be a `JedisAccessControlException` which would extend `JedisDataException`. It can be used directly for now or sub-classed as JedisAuthenticationException and JedisPermissionException.

Reply:
I have solved that in the commit 439d7aa692cd0ea4116d6307eefac2a76f5ca5f0